implementation/src/test/java/io/smallrye/mutiny/operators/MultiOnItemFailWithTest.java

```diff
@@ -20,8 +21,14 @@
 
 public class MultiOnItemFailWithTest {
 
-    private final MultiOnCancellationSpy<Integer> items = Spy.onCancellation(Multi.createFrom().items(1, 2));
-    private final MultiOnCancellationSpy<Integer> nothing = Spy.onCancellation(Multi.createFrom().nothing());
+    private MultiOnCancellationSpy<Integer> items;
```

Comment:
What does it bring to move their initialization?

Reply:
Spies have counters, so in a per-class test instantiation you don't want to share 😃